msu/systems/registry/load.nut

```diff
@@ -7,17 +7,18 @@ includeFile("registry_system.nut");
 local system = ::MSU.Class.RegistrySystem();
 
 ::MSU.System.Registry <- system;
-::MSU.Mods <- system.Mods;
+
+::MSU.registerMod <- function( _id, _version, _name = null )
```

Comment:
As in a previous comment, we should discuss if we want this wrapper. Keep the discussion on that previous comment to keep it in one place.